example/src/main/scala/example/AuthenticationServer.scala

```diff
@@ -1,13 +1,16 @@
 package example
 
 import pdi.jwt.{Jwt, JwtAlgorithm, JwtClaim}
+import zhttp.http.Middleware.bearerAuth
 import zhttp.http._
 import zhttp.service.Server
 import zio.{App, ExitCode, URIO}
 
 import java.time.Clock
 
-object Authentication extends App {
+object AuthenticationServer extends App {
+  // Start this server and use AuthenticationClient to send requests
```

Comment:
server is missing the URL that will generate the token and send back to client the token in response.

Reply:
The login app is doing that.